apollo-router/tests/integration/traffic_shaping.rs

```diff
@@ -180,7 +188,13 @@ async fn test_router_custom_metric() -> Result<(), BoxError> {
         .telemetry(Telemetry::Otlp { endpoint: None })
         .config(format!(
             r#"
-            {PROMETHEUS_CONFIG}
+            telemetry:
+                exporters:
+                    metrics:
+                        prometheus:
+                            listen: 127.0.0.1:4000
+                            enabled: true
+                            path: /metrics
                 instrumentation:
```

Comment:
This one was especially horrifying:
```yaml
{PROMETHEUS_CONFIG}
    instrumentation:
```
it relies on the indentation inside the `PROMETHEUS_CONFIG` const to be aligned with this string, and you can't know what `instrumentation:` refers to without reading the `PROMETHEUS_CONFIG` value.